### ISSUE TITLE
scratch-messaging: error opening indexedDB

### ISSUE THREAD

**Issue by Ermageeerd** [ScratchAddons/ScratchAddons#7113]
### Description

UnknownError: Internal error opening backing store for indexedDB.open.

### Steps to reproduce

I don't know, it just errored out.

### Demonstration(s) and additional context

![image](https://github.com/ScratchAddons/ScratchAddons/assets/155028731/8ddadc21-e4cc-45d5-ae9d-9b5b99b1dc16)

### Environment

Browser: <!Edge 121.0.2277.83>
Scratch Addons version: <!1.35.1.>
Operating system: <!Windows 11> (just in case)

**Comment by Samq64** [comment]
We've been getting a fair amount of feedback about this but could never repro. I think this is the first time a screenshot was posted.

**Comment by WorldLanguages** [comment]
@Ermageeerd Restarting your browser usually fixes the bug.

**Comment by Hans5958** [comment]
This issue has been raised multiple times privately, but an issue never been created because "We have nothing to do with this, we don't file browser bugs here just because we are affected in some way" (@apple502j)

In any case, I disagree and asked for an issue for everyone to notice. I invite her (or anyone) to explain what may happen.

In the meantime, we can link this issue on the error of the Scratch Addons so more info can be gathered, if that's the concern.

**Comment by Ermageeerd** [comment]
> @Ermageeerd Restarting your browser usually fixes the bug.

Thanks!

**Comment by Ermageeerd** [comment]
bruj i forgot abt this ill close it

**Comment by Hans5958** [comment]
No, this issue is valid. This issue should be closed only if these errors won't appear again.

@WorldLanguages Have we fixed this issue?

**Comment by WorldLanguages** [comment]
It still happens in Chromium from time to time

**Comment by Hans5958** [comment]
Ah, okay, then. So let this issue open.
